app/models/question.rb

```diff
@@ -59,6 +59,11 @@ def self.with_answers
     includes(:answers)
   end
 
+  def self.by_tags(tags_id)
+    tags_id = tags_id.map(&:to_i)
+    with_basic_association.joins(:tags).where(tags: {id: tags_id}).distinct
```

Comment:
With the `tags_id.map` above, does it mean we are expecting the client to send the **tags** as an array. I thought it was coming as comma separated string. And maybe you will put me through on why we need the `distinct` when I see you. 🙏